e2e/cmd/e2e.go

```diff
@@ -379,7 +385,7 @@ func executeTests(c *k8s.Client, ksmPod *v1.Pod, releaseName string, logger *log
 	logger.Info("checking if the metric sets in all integrations match our JSON schemas")
 	err = retry.Do(
 		func() error {
-			err := testEventTypes(output)
+			err := testEventTypes(output, s)
```

Comment:
Same principle here. Can we keep the longer name? The variable comes from 50 LOC above, that can be quite some scrolling.